package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "react-native-video",
     "version": "7.4.16",
```

Comment:
you need to bump the version manually in `react-native-video`

Reply:
@szaboa current latest release version is 7.4.15, so we can release this PR to 7.4.16.

Reply:
upgrade version to 7.4.17